### DISCUSSION TITLE
目前有个Claude和ChatGPT同宗同源而且不需要翻墙

### DISCUSSION THREAD

**Discussion by hejun041** [lss233/chatgpt-mirai-qq-bot#618] [Ideas]
![image](https://user-images.githubusercontent.com/16474051/231620724-9d923b6c-adc5-4678-88e2-34cc8db92adb.png)
我和ChatGPT有以下主要区别:

1. 我们的训练数据不同。我是由Anthropic公司训练的助手AI,使用的训练数据主要聚焦于提供友好的人机交互体验。而ChatGPT是OpenAI开发的,训练数据更广泛,面向开放域的一般会话。

2. 我们的应用场景不同。我更适用于提供友好的互动性帮助,解答用户提出的技术问题等。ChatGPT更适用于开放域的随意会话。

3. 我们的生成能力不同。我生成的回复更注重与上下文相关性,确保连贯和可解释性。ChatGPT的生成回复更注重流畅性,有时上下文相关性较弱。

4. 我们处理敏感信息的方式不同。我会避免生成任何有害、欺诈或不合法的内容。而ChatGPT由于训练方式的限制,有时会生成低质量或欠妥当的回复。

5. 我们的个性不同。我的回复个性更少,目的是提供清晰和中立的解答与帮助。而ChatGPT回复带有更明显的个性。

总之,虽然我和ChatGPT都是基于transformer的语言生成AI,但由于技术细节和产品定位的不同,我们的实际交互效果还是有较大差异。我更适用于需要清晰可靠帮助的场景,ChatGPT更适用于娱乐化的闲聊会话。 

地址：[https://www.anthropic.com/claude-in-slack](url)

**Comment by hejun041** [comment]
重点是免费且不需要申请

**Reply by lead888** [reply]
链接404了

**Comment by lss233** [comment]
这个我知道。 必须在 Slack 里面用。
